code_challenges/code_wars/backspace_string.py

```diff
@@ -0,0 +1,28 @@
+class Backspace:
+    def __init__(self, characters):
+        self.characters = characters
+
+    def hit_backspace(self):
+        while "#" in self.characters:
+            for i, character in enumerate(self.characters):
+                if character == "#":
```

Comment:
Can you replace this `for` loop with self.characters.find("#"). https://docs.python.org/3/library/stdtypes.html#str.find